docs/application/dotnet/guides/nui/ui-components.md

```diff
@@ -22,6 +22,7 @@ The following table lists the available UI components:
 | [RadioButton](./nui-components/RadioButton.md) | A RadioButton that can set selected or unselected status when user selects it. | `RadioButton`                |
 | [Slider](./nui-components/Slider.md) | A slider that indicates a modifiable value within a specific range. | `Slider`                   |
 | [Switch](./nui-components/Switch.md) | A switch that can be used as a selector. | `Switch`                |
+| [Notification](./nui-components/Notification.md) | A Notification is help to raise a notification window with a content View. | `Notification`                |
```

Comment:
Suggest rephrasing to:

A notification is used to pop-up a notification window with a content view.

Also, as the table is in alphabetical order, suggest adding notification after Loading.

Reply:
Done